Userland/Libraries/LibJS/Runtime/ECMAScriptFunctionObject.h

```diff
@@ -153,7 +153,7 @@ class ECMAScriptFunctionObject final : public FunctionObject {
 
     bool m_has_parameter_expressions { false };
     bool m_has_duplicates { false };
-    HashTable<DeprecatedFlyString> m_parameter_names;
+    HashMap<DeprecatedFlyString, bool /* is_local */> m_parameter_names;
```

Comment:
I think a comment explaining what a bool is for is prime time to add a named enum

Reply:
Excellent point! Done :)